style.css

```diff
@@ -439,3 +437,262 @@ footer a {
   margin-bottom: 1%;
   background-color: #0d151e;
 }
+
+.nav-bar a {
+  text-decoration: none;
+  font-family: Inter, sans-serif;
+  font-size: 0.7rem;
+  font-style: normal;
+  font-weight: 600;
+  color: #344563;
+}
+
+.nav-bar {
+  display: none;
+  gap: 2rem;
+}
+
+.image-aboutme-desktop {
+  display: none;
+}
+
+.line {
+  height: 1px;
+  width: 100vw;
+  background-color: #dfe1e6;
+  margin-top: 5%;
+  margin-bottom: -4%;
+  display: none;
+}
+
+.image-container-bottom {
+  display: none;
+  width: 100%;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.img-height {
+  width: 80px;
+  height: 80px;
+}
+
+@media screen and (min-width: 550px) {
+  body {
+    background-size: 100% 10%;
+    background-position-y: 2%;
+  }
+}
+
+@media screen and (min-width: 550px) {
+  body {
+    background-size: 100% 11%;
+  }
+}
```

Comment:
[OPTIONAL]@muhammaduzairrazaq  Good job so far, I would suggest you to do not repeat same thing twice in order to follow the best practices 😎